daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-publish-subscribe.md

```diff
@@ -585,28 +619,34 @@ dapr run --app-id orderprocessing --app-port 6001 --dapr-http-port 3601 --dapr-g
 
 ```javascript
 //dependencies
-import { DaprServer, DaprClient, CommunicationProtocolEnum } from 'dapr-client'; 
+import { DaprServer, CommunicationProtocolEnum } from 'dapr-client'; 
 
 //code
 const daprHost = "127.0.0.1"; 
+const serverHost = "127.0.0.1";
+const serverPort = "6002"; 
```

Comment:
Not a fan of these hard coded values 👎 Shouldn't this be 6001 for the message to be sent correctly given this below?

dapr run --app-id orderprocessing --app-port 6001 --dapr-http-port 3601 --dapr-grpc-port 60001 npm start

Reply:
Also out of interest, do you have working code examples for each of these languages?

Reply:
Yes. I have working code examples for each of these languages.